objects/src/lib.rs

```diff
@@ -61,3 +61,12 @@ pub const ACCOUNT_TREE_DEPTH: u8 = 64;
 
 /// The depth of the Merkle tree used to commit to notes produced in a block.
 pub const NOTE_TREE_DEPTH: u8 = 20;
+
+/// The maximum number of assets that can be stored in a single note.
+pub const MAX_ASSETS_PER_NOTE: usize = 256;
+
+/// The maximum number of notes that can be consumed by a single transaction.
+pub const MAX_INPUT_NOTES_PER_TX: usize = 1023;
```

Comment:
Just curious why is `MAX_INPUT_NOTES_PER_TX` not a power of 2, while most other constants are. 

I think that it is because of how the memory is laid out and how input notes can be processed but didn't find all the associated code that explained this.

Reply:
I'm actually not sure. There may have been some rational for it, but not sure if it is valid any longer. In general, we'll need to update most of these limits (see #398).